internal/powerplatform/resource_billing_policy_test.go

```diff
@@ -470,47 +470,49 @@ func TestUnitTestBillingPolicy_Validate_Create_WithoutFinalStatusInPostResponse(
 	})
 }
 
-func TestUnitTestBillingPolicy_Validate_Create_TimeoutWithoutFinalStatus(t *testing.T) {
-	httpmock.Activate()
-	defer httpmock.DeactivateAndReset()
+// commenting out until we can properly test timeouts
```

Comment:
This probably makes more sense in a long-lived feature branch as opposed to Main